### PR TITLE
Policy enhanced

### PR DESCRIPTION
**Pull request by maheshopsmx**
Removed the old Custom policy and Enhanced it as the Policy stage with more input parameters and dedicated image with more modification as Suggested by Vasantha

### CODE REVIEW

File: charts/oes/charts/spinnaker/config/orca-local-pipeline-promotion.yml [hunk 1]
```diff
@@ -307,8 +286,10 @@ job:
                         value: "${{execution.startTime} .toString()}"
                       - name: user
                         value: "${execution.authentication.user}"
-                    image: 'opsmxdev/apline-with-curl'
+                    image: 'opsmxdev/policy:v1'
```
Comment (abhinaybyrisetty) [inline charts/oes/charts/spinnaker/config/orca-local-pipeline-promotion.yml:289]: Clarification: So, we put the script into the image now?

File: charts/oes/charts/spinnaker/config/orca-local-pipeline-promotion.yml [hunk 2]
```diff
@@ -235,70 +235,49 @@ job:
...
-                        fi
-                        EOL
-                        chmod +x run.sh > /dev/null 2>&1
-                        sh run.sh
-                    command:
-                      - /bin/sh
-                      - '-x'
-                      - '-c'
+                  - args:
+                      -
```
Comment (abhinaybyrisetty) [inline charts/oes/charts/spinnaker/config/orca-local-pipeline-promotion.yml:273]: Why do we need this, if we are not passing any arguments?

File: charts/oes/charts/spinnaker/config/orca-local.yml [hunk 3]
```diff
@@ -1,70 +1,49 @@
...
-                        fi
-                        fi
-                        EOL
-                        chmod +x run.sh > /dev/null 2>&1
-                        sh run.sh
-                    command:
-                      - /bin/sh
-                      - '-x'
-                      - '-c'
+                  - args:
```
Comment (abhinaybyrisetty) [inline charts/oes/charts/spinnaker/config/orca-local.yml:38]: Why do we need this, if we are not passing any arguments?

File: charts/oes/charts/spinnaker/config/orca-local.yml [hunk 4]
```diff
@@ -73,8 +52,10 @@ job:
                         value: "${{execution.startTime} .toString()}"
                       - name: user
                         value: "${execution.authentication.user}"
-                    image: 'opsmxdev/apline-with-curl'
+                    image: 'opsmxdev/policy:v1'
```
Comment (abhinaybyrisetty) [inline charts/oes/charts/spinnaker/config/orca-local.yml:55]: Clarification: So, we put the script into the image now?

File: charts/oes/charts/spinnaker/config/orca-local-pipeline-promotion.yml (hunk 1 above)
Comment (maheshopsmx) [reply charts/oes/charts/spinnaker/config/orca-local-pipeline-promotion.yml:289]: yes Abhinay

File: charts/oes/charts/spinnaker/config/orca-local.yml (hunk 4 above)
Comment (maheshopsmx) [reply charts/oes/charts/spinnaker/config/orca-local.yml:55]: yes, Abhi in order to reduce disturbance in Orca-local.yml

File: charts/oes/charts/spinnaker/config/orca-local-pipeline-promotion.yml (hunk 2 above)
Comment (maheshopsmx) [reply charts/oes/charts/spinnaker/config/orca-local-pipeline-promotion.yml:273]: This args is needed for the orca to pass env variable to script

File: charts/oes/charts/spinnaker/config/orca-local.yml (hunk 3 above)
Comment (maheshopsmx) [reply charts/oes/charts/spinnaker/config/orca-local.yml:38]: This args is needed for the orca to pass env variable to the script

Comment (abhinaybyrisetty) [reply charts/oes/charts/spinnaker/config/orca-local.yml:38]: Ok

**Review by abhinaybyrisetty — APPROVED** [review]

**Pull request merged**